### PR TITLE
Kot/create deskof cards

### PR DESCRIPTION
**Pull request by sergioale9**

### CODE REVIEW

**Review by J0c4 — CHANGES_REQUESTED** [review]
@sergioale9 @AdriOli134 this looks good, please check my comments.

Also, PR #89 from @saraialvarezrosas and @MariaAT18 is approved and ready to merge, you might want to wait for that to be merged and then do a rebase along with the updates I'm requesting for your PR. If you get rebased and updated before #89 is merged, we can go ahead and merge yours, in such case, @saraialvarezrosas will need to do rebase again, if you want, sync by chat how do you want to proceed

File: src/main/java/jalau/at18/kingoftokyo/DeskCards.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package jalau.at18.kingoftokyo;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class DeskCards {
```
[the remark this answers is not in the document]
Comment (sergioale9) [reply src/main/java/jalau/at18/kingoftokyo/DeskCards.java]: changed the name of class and methods

File: src/main/java/jalau/at18/kingoftokyo/DeskCards.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    private List<CompleteCard> deskOfCards;
```
[the remark this answers is not in the document]
Comment (sergioale9) [reply src/main/java/jalau/at18/kingoftokyo/DeskCards.java]: Changed type of object list, from CompleteCard to Card

File: src/test/java/jalau/at18/kingoftokyo/DeskCardsTest.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import org.junit.Test;
+
+public class DeskCardsTest {
+
+    @Test
+    public void deskShouldHaveCards() {
+        DeskCards deskCards = new DeskCards();
+        assertEquals(11, deskCards.getDesk().size());
+
```
[the remark this answers is not in the document]
Comment (sergioale9) [reply src/test/java/jalau/at18/kingoftokyo/DeskCardsTest.java]: solved

**Review by J0c4 — CHANGES_REQUESTED** [review]
Updates look good, thanks.

There are conflicts that need to be solved, please try to solve them using git rebase, if you need help, we can check during tomorrow's class

**Review by J0c4 — APPROVED** [review]

**Pull request merged**